frontend/src/pages/home-page.tsx

```diff
@@ -20,8 +20,8 @@ function HomePage() {
   }, []);
   const navigate = useNavigate();
   return (
-    <div className="font-[Poppins] w-full">
-      <div className="relative h-[460px] bg-cover mt-[-8px] bg-fixed bg-center bg-[url('https://i.ibb.co/Pm4ScWL/wanderlust-bg.webp')]">
+    <div className="font-[Poppins] w-full dark:dark">
+      <div className="relative h-[460px] bg-cover mt-[-8px] bg-fixed bg-center bg-[url('https://get.wallhere.com/photo/landscape-sea-beach-hotel-swimming-pool-resort-tropical-lagoon-Caribbean-vacation-estate-leisure-villa-225624.jpg')]">
```

Comment:
Don't revert the url, let it be same as before

Reply:
sir I han't reverted the URl.I

Reply:
At least I can see it as changed